src/test/java/org/jenkinsci/plugins/workflow/steps/CoreWrapperStepTest.java

```diff
@@ -90,13 +90,20 @@
         story.addStep(new Statement() {
             @Override public void evaluate() throws Throwable {
                 new SnippetizerTester(story.j).assertRoundTrip(new CoreWrapperStep(new MockWrapper()), "mock {\n    // some block\n}");
-                Assume.assumeFalse(Functions.isWindows()); // TODO create Windows equivalent
                 Map<String,String> slaveEnv = new HashMap<>();
-                slaveEnv.put("PATH", "/usr/bin:/bin");
+                if (Functions.isWindows()) {
+                    slaveEnv.put("PATH", "c:\\windows\\System32");
+                } else {
+                    slaveEnv.put("PATH", "/usr/bin:/bin");
+                }
                 slaveEnv.put("HOME", "/home/jenkins");
                 createSpecialEnvSlave(story.j, "slave", "", slaveEnv);
                 WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "p");
-                p.setDefinition(new CpsFlowDefinition("node('slave') {mock {semaphore 'restarting'; echo \"groovy PATH=${env.PATH}:\"; sh 'echo shell PATH=$PATH:'}}", true));
+                if (Functions.isWindows()) {
+                    p.setDefinition(new CpsFlowDefinition("node('slave') {mock {semaphore 'restarting'; echo \"groovy PATH=${env.PATH}:\"; bat 'echo shell PATH=%PATH%:'}}", true));
+                } else {
+                    p.setDefinition(new CpsFlowDefinition("node('slave') {mock {semaphore 'restarting'; echo \"groovy PATH=${env.PATH}:\"; sh 'echo shell PATH=$PATH:'}}", true));
+                }
```

Comment:
nit: I would reduce the duplication here a bit:

```suggestion
                p.setDefinition(new CpsFlowDefinition("node('slave') {mock {semaphore 'restarting'; echo \"groovy PATH=${env.PATH}:\"; " + (Functions.isWindows()
                        ? "bat 'echo shell PATH=%PATH%:'}}"
                        : "sh 'echo shell PATH=$PATH:'}}"), true));
```